pythonfile/flask_api.py

```diff
@@ -0,0 +1,87 @@
+
+from flask import Flask, request, jsonify
+from pymongo import MongoClient
+from manifest_validator import ManifestValidator
+import time, json5, uuid, json
+from first import jsonstring
+import elastic
+from elasticsearch import Elasticsearch
+
+client=MongoClient('mongodb:27017')
+db=client.knexDBmh1
+coll=db.projects
+
+es = Elasticsearch(['http://elasticsearch:9200'])
+
+
+app=Flask(__name__)
+
+
+
+_id=uuid.uuid4()
+
+
+# add project to test with
+@app.before_first_request

```

Comment:
Well, I don't know how. I don't see it, when I try to edit it. Even if I write it again, it doesn't see the changes